lib/sensors/INA_226.cpp

```diff
@@ -83,6 +83,19 @@ int INA_226::configureSensor(SensorModes configuration_bits)
     return 0;
 }
 
+u_int16_t INA_226::readConfigRegister()
+{
+    char cmd[2] = {CONFIG_REGISTER, 0x00};
+    u_int16_t registerData = 0x00;

```

Comment:
Tiny nitpick, but does this variable actually need to be instantiated before you set it on line 94?